index.html

```diff
@@ -3743,55 +3753,50 @@ <h2>DID Resolution</h2>
 be a conformant <a>DID</a> as defined in <a href="#did-syntax"></a>.
             </dd>
             <dt>
-did-resolution-input-metadata
+options
             </dt>
             <dd>
-A <a href="#metadata-structure">metadata structure</a> consisting of input
-options to the <code>resolve</code> and <code>resolveRepresentation</code>
-functions in addition to the <code>did</code> itself. Properties defined by this
-specification are in <a href="#did-resolution-input-metadata-properties"></a>.
-This input is REQUIRED, but the structure MAY be empty.
+A <a href="#metadata-structure">metadata structure</a> containing properties
+defined in <a href="#did-resolution-options"></a>. This input is
+REQUIRED, but the structure MAY be empty.
             </dd>
         </dl>
 
         <p>
-The output variables of these functions are as follows:
+The return value of <code>resolve()</code> is <dfn>Resolution</dfn> which is
```

Comment:
As we discussed on yesterday's [2021-02-02 Topic Call](https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-02-02-did-topic), the resolution and dereferencing functions really return three separate values, rather than a single map with three properties. When you dereference an HTTP URL, the HTTP body (representation of a resource) and the HTTP headers are also separate things. [RFC3986](https://tools.ietf.org/html/rfc3986#section-1.2.2) says:

> the most common form of URI dereference is "retrieval": making use of a URI in order to retrieve a representation of its associated resource [...] additional information might be supplied about the resource (resource metadata)

Having said that, concrete implementations e.g. in TypeScript may choose to return a single JSON object that contains all three return values, and we could add this as an example, similar to how in the DID Resolution spec we define a [JSON-LD structure](https://w3c-ccg.github.io/did-resolution/#example) that can hold all three return values.